packages/shared-base/tests/utils/ABTest.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'vitest'
+import { joinsABTest, getABTestSeed } from '../../src/utils/ABTest.js'
+
+describe('detect if an installation joins the A/B test', () => {
+    it('should calculate join A/B test correctly', () => {
+        expect(joinsABTest(100)).toBeTruthy()
+        expect(joinsABTest(0)).toBeFalsy()
```

Comment:
Still have `1%` chance to be truthy.

Reply:
Once failed https://github.com/DimensionDev/Maskbook/actions/runs/4977525342/jobs/8906690436?pr=9600

Reply:
already fixed!